primitives/io/src/lib.rs

```diff
@@ -712,7 +712,7 @@ pub trait Offchain {
 		deadline: Option<Timestamp>,
 	) -> Result<u32, HttpError> {
 		self.extension::<OffchainExt>()
-			.expect("http_response_read_body can be called only in the offchain worker context")
```

Comment:
Are these needed?

Reply:
Nah, it's already in master, just got squashed